packages/server/src/server.ts

```diff
@@ -134,11 +135,48 @@ app.getKoaApp().use(async (ctx, next) => {
   }
 });
 
+app.getKoaApp().use(async (ctx, next) => {
```

Comment:
Let's look to rework this method next sprint.
Could see readability and testability improvements

Reply:
Okay